app/ui/static/css/app.css

```
@@ -0,0 +1,856 @@
+
+.modal-footer {
+  padding: 14px 20px;
+  background: var(--v-bg-subtle);
+  border-top: 1px solid var(--v-border-subtle);
+  display: flex;
+  align-items: center;
+  justify-content: flex-end;
+  gap: 10px;
+}
+
+@keyframes modalFadeIn {
```

Stylelint (17.14.0)

[error] Expected keyframe name "modalFadeIn" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)